share/man/man4/rtw89.4

```diff
@@ -100,23 +105,21 @@ The
 .Nm
 driver supports PCIe devices with the following chipsets:
 .Pp
-.Bl -tag -width Ds -offset indent -compact
-.It Realtek 8851BE Wi-Fi 6  (RTL8851BE)
-.It Realtek 8852AE Wi-Fi 6  (RTL8852AE)
-.It Realtek 8852BE Wi-Fi 6  (RTL8852BE)
-.It Realtek 8852CE Wi-Fi 6E (RTL8852CE)
-.It Realtek 8922AE Wi-Fi 7  (RTL8922AE)
+.Bl -bullet -compact
+.It
+Realtek 8851BE Wi-Fi 6  (RTL8851BE)
+.It
+Realtek 8852AE Wi-Fi 6  (RTL8852AE)
+.It
+Realtek 8852BE Wi-Fi 6  (RTL8852BE)
+.It
+Realtek 8852CE Wi-Fi 6E (RTL8852CE)
+.It
+Realtek 8922AE Wi-Fi 7  (RTL8922AE)
 .El
```

Comment:
This is a duplicate of the HARDWARE section, please remove from `.Sh HARDWARE` at line 103.
So a deletion in the following range: 103-119

Reply:
Thanks